config/pass.js

```diff
@@ -24,6 +24,46 @@ module.exports = function(passport, LocalStrategy) {
     }
   ));
 
+  // Determine if using production URL or localhost
+  if (process.env.NODE_ENV === 'production') {
+    var clientID = process.env.FACEBOOK_APP_ID;
+    var clientSecret = process.env.FACEBOOK_APP_SECRET;
+  } else {
+    var clientID = process.env.FACEBOOK_DEV_APP_ID;
+    var clientSecret = process.env.FACEBOOK_DEV_APP_SECRET;
+  }
+
+  passport.use(new FacebookTokenStrategy({
+      clientID: clientID,
+      clientSecret: clientSecret
+    },
+    function(accessToken, refreshToken, profile, done) {
+      User.findOne({ facebookId: profile.id }, function(err, user) {
+        if (err) {
+          return done(err);
+        }
+
+        // If the user does not exist, then create it
+        if (!user) {
```

Comment:
Create a new user if it doesn't exist. 

Attempt to get the user's firstname by splitting the display name field.
Attempt to get the first email in the facebook user's list of emails
Verify the user already, no email verification required